### DISCUSSION TITLE
Clicking on Algolia search result does not bring me to the exact location, using docsearch API

### DISCUSSION THREAD

**Discussion by SeeYangZhi** [vuepress/core#626] [Q&A] [answered]
I have implemented the docsearch plugin but clicking on the search result on the webpage does not navigate the page to the location. The URL will be of the search result for 1 sec but jump back to the previous URL almost instantly.

To replicate the issue: https://seeyangzhi.github.io/Tools-Archive and search for something and click on the result.

UPDATE#1:
* It works if you hit enter instead of clicking on the search result.

### Configs:

I am using v2.0.0-beta.33 for all plugins and using the default theme. 

My CONFIG for the crawler:
```
{
  "index_name": "Tools-Archive",
  "start_urls": ["https://seeyangzhi.github.io/Tools-Archive/"],
  "stop_urls": [],
  "selectors": {
    "lvl0": {
      "selector": "p.sidebar-heading.active",
      "global": true,
      "default_value": "Documentation"
    },
    "lvl1": ".theme-default-content h1",
    "lvl2": ".theme-default-content h2",
    "lvl3": ".theme-default-content h3",
    "lvl4": ".theme-default-content h4",
    "lvl5": ".theme-default-content h5",
    "text": ".theme-default-content p, .theme-default-content li",
    "lang": {
      "selector": "/html/@lang",
      "type": "xpath",
      "global": true
    }
  },
  "strip_chars": " .,;:#",
  "custom_settings": {
    "attributesForFaceting": ["lang", "tags"]
  }
}
```

**Comment by Mister-Hope** [comment, marked as answer]
Hey, try updating to newest version, if the issue still exist, open a minial reproduction repo. I doubt there are any issues in docsearch plugin.

Also, you should better move to algolia crawler.

See https://github.com/vuepress/vuepress-next/pull/721 for related docs (it's not been approved yet)
